src/main/resources/application.yml

```diff
@@ -1,94 +1,100 @@
 # SERVER ENDPOINT
 server:
-   host: ${SERVER_HOST:localhost}
-   port: ${SERVER_PORT:8080}
-   servlet:
-     context-path: ${SERVER_CONTEXT:/}
-   tomcat:
-     remoteip:
+  host: ${SERVER_HOST:localhost}
```

Comment:
clean edits on this file, the only modification should be the addition of webauthn block